ddp-workspace/projects/toolkit/src/lib/components/dashboard/dashboard.component.spec.ts

```diff
@@ -64,4 +76,41 @@ describe('DashboardComponent', () => {
         const subjectPanel = fixture.debugElement.query(By.css('ddp-subject-panel'));
         expect(subjectPanel).toBeTruthy();
     });
+
+    it('should reset participant in session on init for participant dashboard', () => {
+        toolkitConfigMock.useParticipantDashboard = true;
+        component.ngOnInit();
+        fixture.detectChanges();
+
+        expect(sessionSpy.setParticipant).toHaveBeenCalledWith(null);
+    });
+
+    it('should not reset participant in session on init for regular dashboard', () => {
+        toolkitConfigMock.useParticipantDashboard = false;
+        component.ngOnInit();
+        fixture.detectChanges();
+
+        expect(sessionSpy.setParticipant).not.toHaveBeenCalled();
+    });
+
+    it('should set participant in session before action navigation for participant dashboard', () => {
+        toolkitConfigMock.useParticipantDashboard = true;
+        component.ngOnInit();
+        fixture.detectChanges();
+
+        const participantGuid = 'userGuid567';
+        component.navigate('1', participantGuid);
```

Comment:
Would be nice to have a named constant for '1' value.

Reply:
why? this number is totally random and might be any other string

Reply:
It seems like a `magic` value.

Reply:
it's a test, it's ok using random string like '123', we don't check anywhere that this number was passed correclty means can pass anything